src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.container.js

```diff
@@ -74,13 +75,17 @@ export const EnrollmentPageDefault = () => {
     const onEventClick = (eventId: string) => {
         history.push(`/enrollmentEventEdit?${buildUrlQueryString({ orgUnitId, eventId })}`);
     };
+
     const onUpdateTeiAttributeValues = useCallback((updatedAttributeValues, teiDisplayName) => {
         dispatch(updateEnrollmentAttributeValues(updatedAttributeValues
             .map(({ attribute, value }) => ({ id: attribute, value })),
         ));
         dispatch(updateTeiDisplayName(teiDisplayName));
     }, [dispatch]);
 
+    const onUpdateEnrollmentDate = useCallback(enrollmentDate =>
+        dispatch(updateEnrollmentDate(enrollmentDate)), [dispatch]);
```

Comment:
Hey @superskip,
IMHO the enrollment date in the top ScopeSelector should be updated without requiring a page refresh. The `WidgetProfile` currently handles it in`onUpdateTeiAttributeValues` and `updateTeiDisplayName(teiDisplayName)`. Does it make sense to implement something similar for the enrollment date? 

https://github.com/dhis2/capture-app/assets/6817073/e0ce501e-bacd-45e1-b62b-db550a519d80

Thanks you!

Reply:
Good observation! Will fix :)